packages/sitecore-jss/src/site/graphql-siteinfo-service.ts

```diff
@@ -150,8 +191,27 @@ export class GraphQLSiteInfoService {
       hasNext = response.search.pageInfo.hasNext;
       after = response.search.pageInfo.endCursor;
     }
+    return results;
+  }
 
-    this.cache.setCacheValue(this.getCacheKey(), results);
+  protected async fetchWithSiteQuery(): Promise<SiteInfo[]> {
+    const results: SiteInfo[] = [];
```

Comment:
Can we remove _results_ variable and just return _result_ that is defined on line 203